src/tutorials/0008-introduction-to-libp2p/02.md

```diff
@@ -0,0 +1,46 @@
+---
+    title: "The current problem with Peer-to-Peer Protocols"
+    type: "multiple-choice"
+---
+
+IPFS is not the first attempt at creating a peer-to-peer protocol. Peer-to-peer concepts and implementations have been around for decades.
```

Comment:
> Did you mean libp2p rather than IPFS here? If not, why?

I meant `IPFS` because we're still talking about how IPFS tried to solve all of these issues. I start mentioning libp2p from lesson 3 onwards.
Does it make more sense to use libp2p throughout in this lesson?

Reply:
> Can you please define what peer-to-peer means before digging into the rest of this lesson? And what a peer is (human vs device, etc.)

👍 

Going to add the two definitions here for us to discuss and then I'll push the changes.

### Peer-to-peer

Peer-to-peer networks are decentralised applications that share the workloads between peers. So in IPFS, each peer might contain parts or full files to be shared with other peers. If a peer requests files, any peer that has chunks of the files can participate in the sending requested files. And then, the peer who request the data can later on share the data with other peers.

![image](https://user-images.githubusercontent.com/2353186/101664823-349fb100-3a44-11eb-9301-e8cc6843586d.png)
![image](https://user-images.githubusercontent.com/2353186/101664849-39fcfb80-3a44-11eb-97a2-494e6fa76baf.png)


### Peer

A participant in a decentralised network. Peers are equally privileged, equipotent participants in the application.

Reply:
done! f8555cf8cef515336db69561efd8f124492a6929

Thank you for both your feedback 🙏